### ISSUE TITLE
Search result titles link incorrectly when search is initiated from a subdirectory

### ISSUE THREAD

**Issue by vaughnteegarden** [justinwalsh/daux.io#383]
version 0.1.0

Consider a static page hosted at the url below, which comes up in search results: http://test.com/static/Merchant_Integrator/Merchant_Integration_API/Goals_of_Integration.html

If I initiate the search from the page: http://test.com/static/Merchant_Integrator/Merchant_Integration_API/index.html

the href of the TITLE link is incorrect:
`
<a href="Merchant_Integrator/Merchant_Integration_API/Goals_of_Integration.html">...</a>
`
while the href of the URL link is correct:
`
<a href="../../Merchant_Integrator/Merchant_Integration_API/Goals_of_Integration.html">...</a>
`

Note the TITLE link is lacking the back path (../../)

**Comment by samumist** [comment]
I'm also having the same problem with incorrect search title link when search was initiated from a sub page.

**Comment by onigoetz** [comment]
Hi, I'll have a look at this this week, sorry for the delay

**Comment by onigoetz** [comment]
Hi, I checked and I confirm that links from subfolders were broken.

I fixed them and made a cleaner integration of the search system.

Everything is pushed to the  development branch, can you have a look please ?

**Comment by samumist** [comment]
@onigoetz Many thanks for your work.

I've cloned a fresh copy from the repo and generated a static site of the project's docs with the development branch.

It seems the problem was `half resolved`. The search initiated from the odd-subpage is still broken as before (i.e. heading leads to a 404 not found page) while search generated from the even-subpage has indeed solved the problem.

For example, if I start a search from the `Features/Auto_Syntax_highlight` page (odd page), the search result is shown in the content page, the result's heading link is still broken.

If I start a search from the `Features/Front_matter` page (even page), the search result is shown in an overlay (lightbox) style, and the result's heading link is correct.

I hope this is useful and thanks again for the great work.

**Comment by onigoetz** [comment]
Hello,
I'm sorry I can't understand what you mean by "odd" and "even" pages.

I tried to make the search from the page you specify and in both cases I get the lightbox with the search results.

Maybe there is something with your browser's cache ?

**Comment by samumist** [comment]
@onigoetz 
I tried clean the browser's cache and the problem is still there but a bit different.

Search on certain page won't work at all, for example, the `Feature/Auto_Syntax_Highlight.html`. When type in the search word and hit enter, nothing happens.

A look at the browser's console I found these error messages:

![screenshot 2016-08-06 09 03 01](https://cloud.githubusercontent.com/assets/3883086/17453808/04affc7e-5bb5-11e6-87ea-3f64bda96d49.jpg)
The following code is found at the bottom of `Feature/Auto_Syntax_Highlight.html` page.

```
            <!-- Tipue Search -->
        <script type="text/javascript" src="../tipuesearch/tipuesearch_set.js"></script>
        <script type="text/javascript" src="../tipuesearch/tipuesearch.min.js"></script>

        <script>
            window.onunload = function(){}; // force $(document).ready to be called on back/forward navigation in firefox
            $(document).ready(function() {
                $('#tipue_search_input').tipuesearch({
```

and there's no `tipusesearch_set.js` and `tipuesearch.min.js` found in the generated `tipusesearch` directory.

The generated `tipusesearch` directory is as following:

```
├── tipuesearch.css
├── tipuesearch.js
└── tipuesearch_content.json
```

For the correct pages, the js declaration at the bottom of the html page looks like this:
```
    <!-- JS -->
    
    <script src="../themes/daux/js/daux.js"></script>

            <!-- Tipue Search -->
        <script type="text/javascript" src="../tipuesearch/tipuesearch.js"></script>

```

Some info about my environment and process
Mac OSX 10.11.6
php 5.6.24

The process I generate the static document:
1. clone a fresh copy of the project
2. switch to the develop branch
3. run `./generate` to generate the static html docs
4. copy `static` to my server directory and view it in browser
